wunderio/client-UK-FSA-beta
# FSA-1017: Open first search filter on desktop

To test, go to the following pages locally:
- https://beta.food.gov.uk/search/news-alerts?keywords=research
- https://beta.food.gov.uk/news-alerts/search/news
- https://beta.food.gov.uk/news-alerts/search/alerts
- https://beta.food.gov.uk/news-alerts/search/consultations
- https://beta.food.gov.uk/search/guidance?keywords=research
- https://beta.food.gov.uk/search/research?keywords=research

The first search filter should open automatically at desktop resolutions (breakpoint >= 'md'), but not mobile.
